code/_globalvars/lists/xenobiology.dm

```diff
@@ -0,0 +1,14 @@
+///These global lists exist to allow our element to have weight tables without having to be seperate instances.
+
+///Assoc list of cell line define | assoc list of datum | cell_line
+GLOBAL_LIST_INIT(cell_line_tables, /list/list(

```

Comment:
```suggestion
GLOBAL_LIST_INIT(cell_line_tables, list(

```

Reply:
ok i will commit this but it'll throw a liner error that I couldnt fix :^(